src/SPC/builder/linux/library/openssl.php

```diff
@@ -58,6 +58,12 @@ public function build(): void
             $zlib_extra = '';
         }
 
+        if ($this->builder->getOption('no-strip')) {
+            $debug_extra = '-d ';
```

Comment:
This actually completely disables the optimization (sets `-O0`) so it might be better to just use `-g ` which passes the flag.

Reply:
So https://github.com/crazywhalecc/static-php-cli/pull/379 will definitely be better.